src/models/Project/addLocaleToProject/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import Project from '@models/Project'
+import Locale from '@models/Locale'
+import type { ValidationError } from '@types/ValidationError'
+
+export type AddLocaleToProjectArgs = {
+  projectId: string,
+  localeId: string,
+  userId: string
+}
+
+type AddProjectToLocaleResponse = {
+  project?: Project,
+  errors: Array<ValidationError>
+}
+
+const addLocaleToProject = async ({
+  projectId,
+  localeId,
+  userId
+}: AddLocaleToProjectArgs): Promise<AddProjectToLocaleResponse> => {
+  const errors: Array<ValidationError> = []
+
+  const project = await Project.query()
+    .eager('locales')
```

Comment:
This loads the `locales` field while fetching the initial object (otherwise query just returns the object and not its associations).

Maybe we could find a way to filter the eager fields to load only the requested fields (useful if we want to use this in different contexts without pressuring the db)